# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lirc/linux-2.6.17.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur lirc-0.8.0-orig/drivers/lirc_gpio/lirc_gpio.c lirc-0.8.0/drivers/lirc_gpio/lirc_gpio.c
--- lirc-0.8.0-orig/drivers/lirc_gpio/lirc_gpio.c	2006-01-01 13:43:46.000000000 +0000
+++ lirc-0.8.0/drivers/lirc_gpio/lirc_gpio.c	2006-07-01 17:06:44.000000000 +0000
@@ -48,9 +48,12 @@
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,4,0)
 #include "../drivers/char/bttv.h"
 #include "../drivers/char/bttvp.h"
-#else
+#elif LINUX_VERSION_CODE < KERNEL_VERSION(2,6,17)
 #include "../drivers/media/video/bttv.h"
 #include "../drivers/media/video/bttvp.h"
+#else
+#include "../drivers/media/video/bt8xx/bttv.h"
+#include "../drivers/media/video/bt8xx/bttvp.h"
 #endif
 
 #if BTTV_VERSION_CODE < KERNEL_VERSION(0,7,45)
